core/stat/base/leap_array.go

```diff
@@ -225,7 +225,7 @@ func (la *LeapArray) ValuesConditional(now uint64, predicate base.TimePredicate)
 	if now <= 0 {
 		return make([]*BucketWrap, 0)
 	}
-	ret := make([]*BucketWrap, 0)
+	ret := make([]*BucketWrap, 0, 8)
```

Comment:
maybe la.array.length is better?

Reply:
Good advice. Length of result mostly is same to la.array.length.